tests/e2e/stories-editor/utils.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * Utils file for stories-editor tests.
+ */
+
+/**
+ * Opens the inserter, searches the block and inserts it.
+ * Default helpers from WP won't work due to changed selectors.
```

Comment:
Which helpers in particular and which selectors changed?

Reply:
`insertBlock` helper which uses as `.edit-post-header [aria-label="Add block"]`. Will update the comment, too.

Reply:
We should suggest an upstream change to make this selector more robust.

`[aria-label="Add block"]` doesn't even work in a localized UI...

Reply:
Looks like it's assumed that the tests are always run in a non-localized environment, `aria-label` is used in quite a lot of e2e tests in Gutenberg, probably because mostly the buttons etc. don't really have another differentiator. Not sure what could be an alternative in those cases.

Reply:
Yeah that's a bad assumption from an i18n perspective. Will report this.

Reply:
Opened https://github.com/WordPress/gutenberg/issues/16675.